src/webserver.js

```diff
@@ -145,7 +145,7 @@ function setupExpressApp(app) {
 
 	configureBodyParser(app);
 
-	app.use(cookieParser());
+	app.use(cookieParser(nconf.get('secret')));
```

Comment:
Not sure why we didn't do this before, this allows cookies to be signed, so I can reference them in `req.signedCookies` instead.